package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
Yeah for some reason `npmgitdev list point-geometry` only shows the dependency directly from `terriajs`, not the one from `vector-tile`, despite the fact that `point-geometry` is clearly in `vector-tile`'s package.json.  So basically `npm list` doesn't work right.  Shocking, I know.